web_src/js/features/repo-diff.js

```diff
@@ -89,6 +90,13 @@ export function initRepoDiffConversationNav() {
   });
 }
 
+// Will be called when the show more (files) button has been pressed
+function onShowMoreFiles() {
+  initRepoIssueContentHistory();
+  initViewedCheckboxListenerFor(document);
```

Comment:
`document` seems not necessary. `window.document` is always accessible.

Reply:
I'm not quite sure what you mean...
Do you simply want me to replace `document` with `window.document`?
What's the difference?

Reply:
I mean why do you need to pass it as an argument? `initViewedCheckboxListenerFor()` should be enough, inside `initViewedCheckboxListenerFor` it can access `document` directly.

Reply:
Ah, the reason for that is prior to adding the `data-has-listener` lock, I thought I would need to start from a given HTML element when dynamically adding files.
With the current implementation, that can indeed be done.